src/unity/python/turicreate/visualization/_plot.py

```diff
@@ -159,24 +160,36 @@ def save(self, filepath):
                 _json.dump(spec, fp)
             dirname = _os.path.dirname(__file__)
             relative_path_to_vg2png_vg2svg = "../vg2" + extension
-            absolute_path_to_vg2png_vg2svg = _os.path.join(dirname, 
+            absolute_path_to_vg2png_vg2svg = _os.path.join(dirname,
                 relative_path_to_vg2png_vg2svg)
-            (exitcode, stdout, stderr) = _run_cmdline("node " + 
-                absolute_path_to_vg2png_vg2svg + " " 
+            (exitcode, stdout, stderr) = _run_cmdline("node " +
+                absolute_path_to_vg2png_vg2svg + " "
                 + temp_file_path + " " + filepath)
+
+            print(exitcode)
+            print(stdout)
+            print(stderr)
+
             if exitcode == _NODE_NOT_FOUND_ERROR_CODE:
                 # user doesn't have node installed
                 raise RuntimeError("Node.js not found. Saving as PNG and SVG" +
                     " requires Node.js, please download and install Node.js " +
                     "from here and try again: https://nodejs.org/en/download/")
+
+            if exitcode == _CANVAS_PREBUILT_NON_EXISTENT:
+                raise RuntimeError("Canvas-Prebuilt not found. Saving as PNG and SVG" +
```

Comment:
Let's keep to the naming convention of the package itself (all lowercase, `canvas-prebuilt`).